src/pages/GivePage/MyGiveRow.tsx

```diff
@@ -47,10 +47,18 @@ export const MyGiveRow = ({
     }
   }, [selected, lastSelected]);
   return (
-    <Button
+    <Box
+      tabIndex={0}
       onClick={openEpochStatement}
+      onKeyDown={e => {
```

Comment:
Did you mean to have these changes in this pr? I notice you didn't mention them in the description

Reply:
Yes.  In order to not have nested buttons, I needed to convert this to a div.  Sadly, onClick is not captured on a div when using keyboard... so needed the onkeydown.  and using onkeydown had some strange behavior for the drawer it opened, hence the preventDefault and stopPropagation.  Kind of gnarly